### ISSUE TITLE
Your jenkins host despises me. please halp.

### ISSUE THREAD

**Issue by plaguewolf** [micdoodle8/Galacticraft#2394]
ok, seriously this is getting too annoying for words. ive downloaded GalacticraftCore-1.7-3.0.12.465.jar 50+ times, not even exaggerating. over 50 times. and every. single. time. it gets to 40-60% done and then end of streams. ive downloaded through the website, doing the whole, click, wait 10 seconds, redirect, download thing. ive downloaded direct from jenkins, ive even done wget/curl.

why does you host hate me so much...?! FUUUUU!

TL:DR i can please has alt download link?

sigh. i dont even wanna play minecraft anymore tonight. thats how frustrating this has been.

**Comment by EzerArch** [comment]
> why does you host hate me so much...?! FUUUUU!

Jenkins sometimes seems to bite everyone's hands.

The alt download link can be found here: https://forum.micdoodle8.com/index.php?threads/i-cant-download-galacticraft-what-to-do.3736/

**Comment by plaguewolf** [comment]
heh, thanks. ive never had that issue before, excepting with GC though. its like whatever web server is sending RST packets midway through downloads. weirdness.
